src/ploneintranet/network/browser/likes.py

```diff
@@ -56,7 +56,11 @@ def action(self):
         return "%s/@@toggle_like" % self.context.absolute_url()
 
     def handle_toggle(self):
-        """Perform the actual like/unlike action."""
+        """
+        Perform the actual like/unlike action.
+        Since this does a db write it cannot be called with a GET.
+        """
+        assert(self.request.get('REQUEST_METHOD') == 'POST')
```

Comment:
I think we should not be using assert statements. They might get ignored. See the docs: https://docs.python.org/2/reference/simple_stmts.html#the-assert-statement